### PR TITLE
Fix/font consistency

### LINKED ISSUE #15703: Make font size consistent across panes
Currently the font size is different for each pane. 

jquery__target_html_elements_with_selectors_using_jquery___freecodecamp

I checked CodePen and they're using 15 px for all of their editors. JSBin is using 14px. I think 15 px is a good tradeoff between readability and compactness. So we should make all non-header text 15px.

### PR DESCRIPTION
**Pull request by AbdelghaniTassi**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue : Closes #15703

#### Description

This PR fixes issue #15703 about font size consistency
Font size now looks consistent and coherent (15px)
I modified some css classes font size to make all text have the same size except task descriptions and titles ....
These modifications should not have any impact on tests because i only modified the css (less) code.

### CODE REVIEW

**Comment by QuincyLarson** [comment]
@AbdelghaniTassi Great job! I noticed that the tests are still 14px:

regular_expressions__match_literal_strings___freecodecamp

Can you also make these 15px for consistency?

File: client/less/main.less [hunk 1]
```diff
@@ -271,10 +271,11 @@ h1, h2, h3, h4, h5, h6, p, li {
 
 a {
   font-weight: bold;
+  font-size : 15px;
 }
 
 p {
-  font-size: 16px;
+  font-size: 15px;
 }
```
Comment (QuincyLarson) [inline client/less/main.less]: @AbdelghaniTassi Great job! I noticed that the tests are still 14px:

regular_expressions__match_literal_strings___freecodecamp

Can you also make these 15px for consistency?

Comment (AbdelghaniTassi) [inline client/less/main.less]: @QuincyLarson thanks, the font size of tests is now set to 15px

**Comment by dhcodes** [comment]
Tested locally and **LGTM**. Thanks @AbdelghaniTassi for making this change. Guess I'll need to zoom to 110% though (old eyes) 😄 👍 💯 🌮

**Pull request merged**